libbeat/common/kubernetes/metadata/metadata.go

```diff
@@ -61,10 +61,10 @@ func GetPodMetaGen(
 	metaConf *AddResourceMetadataConfig) MetaGen {
 
 	var nodeMetaGen, namespaceMetaGen MetaGen
-	if nodeWatcher != nil {
+	if nodeWatcher != nil && metaConf.Node.Enabled() {
```

Comment:
Can you also update the docs at https://github.com/elastic/beats/blob/e9df1af4819717ab53c6fc95752c72e87c09b850/libbeat/processors/add_kubernetes_metadata/docs/add_kubernetes_metadata.asciidoc#L123 and https://github.com/elastic/beats/blob/e9df1af4819717ab53c6fc95752c72e87c09b850/libbeat/processors/add_kubernetes_metadata/docs/add_kubernetes_metadata.asciidoc#L123 to leave a note that this setting can be disabled?

Reply:
done

Reply:
thanks!